### ISSUE TITLE
Could not resolve prop-types from OutboundLink

### ISSUE THREAD

**Issue by jhirn** [react-ga/react-ga#511]
Hello. I'm getting an error with could not resolve `prop-types` in my build. This occurred after upgrading a number of packages so I'm assuming it may have been a dependency from another package but I would expect this package to install the necessary dependencies. 

I was able to resolve this by installing prop-types directly but I didn't need it until recently. It doesn't work as a peer dependency either. 

Please let me know if there is additional information I can provide. Here's my package.json deps

```json
  "dependencies": {
    "@mantine/core": "^3.0.5",
    "@mantine/hooks": "^3.0.5",
    "@metamask/onboarding": "^1.0.1",
    "@react-hook/window-size": "^3.0.7",
    "@tailwindcss/aspect-ratio": "^0.3.0",
    "esbuild": "^0.13.9",
    "esbuild-plugin-svgr": "^0.0.3",
    "events": "^3.3.0",
    "glob": "^7.2.0",
    "global": "^4.4.0",
    "inflection": "^1.13.1",
    "postcss": "^8.0.9",
    "postcss-cli": "^9.0.1",
    "postcss-import": "^14.0.2",
    "prop-types": "^15.7.2",
    "react": "^17.0.2",
    "react-dom": "^17.0.2",
    "react-ga": "^3.3.0",
    "react-router-dom": "^6.0.0",
    "react-toastify": "^8.0.3",
    "tailwindcss": "^2.2.17"
  }
```

**Comment by david-potgieter** [comment]
`prop-types` is a [peer dependency](https://github.com/react-ga/react-ga/blob/master/package.json#L40), so just add it:
`yarn add prop-types`, `npm install prop-types`.

**Comment by jhirn** [comment]
> `prop-types` is a [peer dependency](https://github.com/react-ga/react-ga/blob/master/package.json#L40), so just add it

Thanks @david-potgieter. This is what I did, but shouldn't it be listed as a dependency of this package so I do not have to do this?

**Comment by Gtstride** [comment]
> `prop-types` is a [peer dependency](https://github.com/react-ga/react-ga/blob/master/package.json#L40), so just add it: `yarn add prop-types` or `npm install prop-types` - whatever your flavour is.

This works for me seemlessly. 🤝

**Comment by PiTiLeZarD** [comment]
I strongly agree with @jhirn here, I came here for the same issue with a different origin, we shouldn't have to add dependencies to our project that aren't directly used by our project, libraries should carry their own dependencies, isn't that the point of a package manager? yes adding this works, no it's not a solution, its a palliative!
